packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1982,12 +1982,13 @@ export class ContainerRuntime
 					clientSignalSequenceNumber: envelope.clientSignalSequenceNumber,
 				});
 			} else if (
-				this.consecutiveReconnects === 0 &&
 				envelope.clientSignalSequenceNumber ===
-					this._perfSignalData.trackingSignalSequenceNumber
+				this._perfSignalData.trackingSignalSequenceNumber
```

Comment:
I am curious, why was this change added by https://github.com/microsoft/FluidFramework/pull/14225?

Reply:
It was actually by mistake - it was not even related to that bug - I must have had that file checked out and missed it